mediachain/reader/api.py

```diff
@@ -56,12 +56,24 @@ def with_fallback():
     return with_thumb
 
 
+def merge(a, b, path=None):
```

Comment:
@autoencoder I think the rules (still super simplified but approaching something, you can imagine how this would work with sort by reputation instead of time) are like this

- primitive (string/number) fields override earlier value
- lists added together as set union
- explicit nulls/empty values clear field
- objects merged recursively following above rules